src/components/Sort/Sort.tsx

```diff
@@ -4,7 +4,7 @@ import styles from '@/components/Sort/Sort.module.css';
 
 const Sort = () => (
     <div className={styles.sortBlock}>
-        <p className={styles.sortText}>Sort by :</p>
+        <p className={styles.sortText}>Sort by:</p>
         <select className={styles.sortSelect}>
```

Comment:
Consider rewriting it like in the example bellow

https://www.w3schools.com/howto/howto_css_dropdown.asp